src/metabase/query_processor/card.clj

```diff
@@ -241,6 +241,14 @@
     (qp.streaming/streaming-response [rff export-format (u/slugify (:card-name info))]
       (qp (update query :info merge info) rff))))
 
+(defn- enrich-parameters-from-card
+  "Allow the FE to omit type and target for parameters by adding them from the card."
+  [parameters card-parameters]
+  (let [id->card-param (->> card-parameters
+                            (map #(select-keys % [:id :type :target]))
```

Comment:
I'm not sure if defaulting the other properties is a good or a bad idea.

Reply:
Seems reasonable to only select the keys the FE used to provide